src/convertToObject.js

```diff
@@ -5,8 +5,23 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  // Remove tab characters and split the string into an array
+  const styleStringArray = sourceString.replaceAll('\t', '').split(';');
+
+  // Reduce the array into an object, trimming the key and value
+  const styleObject = styleStringArray.reduce((acc, style) => {
```

Comment:
u can return it immediately, but up to u)

```suggestion
  return styleStringArray.reduce((acc, style) => {
```